docs/upgrade/1-10-1.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Upgrade to 1.10.1 (Kibana)
+parent: Upgrade
+nav_order: 3
+---
+
+# Upgrade to 1.10.1 (Kibana)
+
+Open Distro for Elasticsearch 1.10.1 includes a breaking change due to Security Kibana Plugin is re-implemented on top of Kibana's new plugin platform.
```

Comment:
"Open Distro for Elasticsearch 1.10.1 includes a breaking change due to the Security Kibana plugin being re-implemented with Kibana's new plugin platform."

Reply:
Resolved